### DISCUSSION TITLE
Can I send a PDF file as an attachment using the Free emailjs ?

### DISCUSSION THREAD

**Discussion by peppermintbunny** [emailjs-com/help#38] [Q&A]
I'm using Vue.js to programmatically send an email with a PDF attachment using emailjs but I get a send error.  Is it possible to send attachments with the free emailjs service?

If it is possible please provide a code example of how to send a PDF file as an attachment.

**Comment by xr0master** [comment]
Attachments to emails and their storage are paid options.

You know, the EmailJS company also pays for this to cloud service companies. Unfortunately, it's nothing free today. We have to say an honest "thank you" to paying customers who sponsor work for all free users.

**Reply by peppermintbunny** [reply]
Thank you for answering.
